crates/index-scheduler/src/snapshots/lib.rs/import_vectors/adding Intel succeeds.snap

```diff
@@ -22,7 +22,7 @@ succeeded [0,1,]
 doggos [0,1,]
 ----------------------------------------------------------------------
 ### Index Mapper:
-doggos: { number_of_documents: 1, field_distribution: {"_vectors": 1, "breed": 1, "doggo": 1, "id": 1} }
```

Comment:
Note: We need to indicate somewhere that `_vectors` was removed from the field distribution